packages/apps-config/src/api/spec/polkafoundry.ts

```diff
@@ -12,7 +12,27 @@ const definitions: OverrideBundleDefinition = {
       // on all versions
       minmax: [0, undefined],
       types: {
-        RelayChainAccountId: 'H256'
+        "RelayChainAccountId": "H256",
+        "RoundIndex": "u32",
+        "SettingStruct": {
+          "bond_duration": "u32",
```

Comment:
CI will complain about these, running `yarn lint` locally should auto-fix to single-quotes